install/pre-install-payload/payload.sh

```diff
@@ -6,8 +6,8 @@ set -o nounset
 
 script_dir=$(dirname "$(readlink -f "$0")")
 
-containerd_repo=${containerd_repo:-"https://github.com/confidential-containers/containerd"}
-containerd_version=${containerd_version:-"v1.6.6.0"}
+coco_containerd_repo=${coco_containerd_repo:-"https://github.com/confidential-containers/containerd"}
+coco_containerd_version=${coco_containerd_version:-"v1.6.6.0"}
```

Comment:
Not related to this change, but whilst you are here, should we update this version to `v1.6.8.2`?

Reply:
Sorry, I saw this had been updated in a later commit, so isn't valid

Reply:
I did! :-)